src/app/dashboard/pools/page.tsx

```
@@ -0,0 +1,26 @@
+"use client";
+import { useEffect, useState } from "react";
+import { PoolCard } from "@/components/PoolCard";
+import { fetchPools } from "@/lib/blend/helpers";
+
+export default function Page() {
+  const [pools, setPools] = useState<any[]>([]);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)